libs/opengl/src/gltext.cpp

```diff
@@ -0,0 +1,268 @@
+/* +------------------------------------------------------------------------+
+   |                     Mobile Robot Programming Toolkit (MRPT)            |
+   |                          https://www.mrpt.org/                         |
+   |                                                                        |
+   | Copyright (c) 2005-2019, Individual contributors, see AUTHORS file     |
+   | See: https://www.mrpt.org/Authors - All rights reserved.               |
+   | Released under BSD License. See: https://www.mrpt.org/License          |
+   +------------------------------------------------------------------------+ */
+
+#include "opengl-precomp.h"  // Precompiled header
+
+// Stripped out version of libCVD gltext.cpp file, ported to OpenGL>=3
```

Comment:
Should Edward Rosten be added to the AUTHORS since this is based on libCVD gltext?

Reply:
:+1:

Done in 43adc0b6342cf8de5aa4e814de15b5a64bc4a8fa